### ISSUE TITLE
A question about `toArray`

### ISSUE THREAD

**Issue by huangdijia** [WendellAdriel/laravel-validated-dto#78]
```php
$user = UserDTO::fromArray([
    'name' => 'deeka',
    'age' => 20,
]);

$user->age = 19;

var_dump($user->toArray()); // output "["name" => 'deeka', "age" => 20]"
var_dump($user->age); // output "19"
```

I want to understand the original design intention of the property value being inconsistent with the values of `toArray`.

**Comment by WendellAdriel** [comment]
That's weird, I'll take a look into this, thanks for the heads up.

**Comment by WendellAdriel** [comment]
Thank you so much for the heads-up! 💪 
The fix was released on v3.6.1!
